build-aci

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 tag" >/dev/stderr
+    exit 1
+fi
+
+tag=$1
+
+tgt=$(mktemp -d)
+
+# Build fleet inside
+docker run -v $PWD:/opt/fleet google/golang:1.3.1 /bin/bash -c "cd /opt/fleet && ./build"
+
+# Generate manifest into target tmp dir
+cat <<DF >${tgt}/manifest
+{
+   "acVersion" : "0.5.1",
```

Comment:
@iaguis sorry I avoided landing this because I wanted to do more
verification that switching fleet to a static build won't break anything.
Did you run the functional tests by chance?

On Tue, Apr 21, 2015 at 9:37 AM, Iago López Galeiras <
notifications@github.com> wrote:

> In build-aci
> <https://github.com/coreos/fleet/pull/1191#discussion_r28796771>:
>
> > +if [ $# -ne 1 ]; then
> > +    echo "Usage: $0 tag" >/dev/stderr
> > +    exit 1
> > +fi
> > +
> > +tag=$1
> > +
> > +tgt=$(mktemp -d)
> > +
> > +# Build fleet inside
> > +docker run -v $PWD:/opt/fleet google/golang:1.3.1 /bin/bash -c "cd /opt/fleet && ./build"
> > +
> > +# Generate manifest into target tmp dir
> > +cat <<DF >${tgt}/manifest
> > +{
> > +   "acVersion" : "0.5.1",
>
> I added validation to the build script. Anything else left to do here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/fleet/pull/1191/files#r28796771>.
>